wordpress-mobile/WordPress-Android
# Fix/en us locale

This PR Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/8910.

The issue is related to the fact that GlotPress doesn't allow to download the original string set as a translation, so we need a different way to get it back.

This PR is based on wordpress-mobile/release-toolkit#16 where the main changes live.
After review and approval of the changes in the release-toolkit, I'll release a new plugin version and I'll update this PR removing the hash reference before merging it.

### To test:
bundle install the new version of the release-toolkit.
run bundle exec fastlane download_promo_strings and verify that the right locale is downloaded for en-US (fast way to test: verify the spelling of favorite in app_store_screenshot2.txt).

### Note:
- This PR also updates the promo strings with the latest content from GlotPress.
- This PR gets rid of the local origin download_metadata action and switches to the shared one.

Update release notes:
